src/AbstractOperations/grid_metrics.jl

```diff
@@ -133,11 +135,59 @@ Adapt.adapt_structure(to, gm::GridMetricOperation{LX, LY, LZ}) where {LX, LY, LZ
 on_architecture(to, gm::GridMetricOperation{LX, LY, LZ}) where {LX, LY, LZ} =
     GridMetricOperation{LX, LY, LZ}(on_architecture(to, gm.metric),
                                     on_architecture(to, gm.grid))
-                                
+
 
 @inline Base.getindex(gm::GridMetricOperation, i, j, k) = gm.metric(i, j, k, gm.grid)
 
 indices(::GridMetricOperation) = default_indices(3)
 
 # Special constructor for BinaryOperation
 GridMetricOperation(L, metric, grid) = GridMetricOperation{L[1], L[2], L[3]}(metric_function(L, metric), grid)
+
+#####
+##### Spacings
+#####
+
+function xspacings(grid, ℓx, ℓy, ℓz)
+    LX, LY, LZ = typeof.((ℓx, ℓy, ℓz))
```

Comment:
```suggestion
    LX, LY, LZ = map(typeof, (ℓx, ℓy, ℓz))
```

broadcasting has a lot of baggage, so its best to avoid it if we don't need the fancy extrusion properties. In this case, we just want to apply a function to a tuple so `map` is more precise and lighter weight

Reply:
Thanks for the suggestion!

Reply:
Maybe more specifically, broadcasting can interfere with type inference whereas there's no issue with `map`. It doesn't matter much for this function specifically but we are generally trying to use `map` when it makes sense throughout the source code

Reply:
Done!